# ANGUS: Analyzing High Throughput Sequencing Data

June 26 - July 8, 2017

Instructors: TBD

[2016 materials:](http://angus.readthedocs.io/en/2016/)
http://angus.readthedocs.io/en/2016/

This intensive two week summer course will introduce attendees with a
strong biology background to the practice of analyzing high-throughput
sequencing data (Illumina, PacBio, and Nanopore).  The first week will
introduce students to computational thinking and large-scale data
analysis on UNIX platforms. The second week will focus on genome and
transcriptome assembly, transcript quantitation, variant calling, and
other topics.

No prior programming experience is required, although familiarity with
some programming concepts is helpful, and bravery in the face of the
unknown is necessary.  2 years or more of graduate school in a
biological science is strongly suggested. Faculty, postdocs, and
research staff are more than welcome, as are industry employees.

Students will gain practical experience in:

- Python and bash shell scripting
- Cloud computing/Amazon EC2
- Basic software installation on UNIX
- Installing and running Trinity, BWA, Salmon, SPAdes, ABySS, Prokka and other bioinformatics tools.
- Querying mappings and evaluating assemblies
- Materials from previous courses are available at http://angus.readthedocs.org/ under a Creative Commons/full use+reuse license.

You can read a blog post about the 2015 course here:
[http://ivory.idyll.org/blog/2015-small-batch.html](http://ivory.idyll.org/blog/2015-small-batch.html)

## [Apply here!](@@)

Applications will close **March 17th**.

----

If you have questions, please contact us at via e-mail at [dibsi.training@gmail.com](dibsi.training@gmail.com).
